backend/PythonClient/multirotor/mission/abstract/abstract_mission.py

```diff
@@ -5,7 +5,7 @@
 
 from PythonClient.multirotor.airsim_application import AirSimApplication
 
-
+#
```

Comment:
What is this change for?

Reply:
Im not too sure what was pushed earlier, but the changes are now there